src/index.js

```diff
@@ -0,0 +1,47 @@
+import './index.css';
+import CheckBoxes from './Checkbox.js';
+import LoadTask from './LoadTask.js';
+import AddTask from './AddTask.js';
+import RemoveTask from './RemoveTask.js';
+
+const toDoList = document.getElementById('todolist');
+
+// calling addTask() when user enters
+const input = document.getElementById('add');
+input.addEventListener('keyup', (event) => {
+  event.preventDefault();
+  if (event.key === 'Enter') {
+    AddTask();
+  }
+});
```

Comment:
You have done a great job✔️😊 adding the 'Keyup' event listener and you also add enter button but haven't added action on this button. It would be best practice to add the action on enter button and another thing that I showed in the picture below. It would be better to reset the input field when the user adds something.
![sawmon](https://user-images.githubusercontent.com/118761303/223354678-3ce3b3f4-e40a-4d0b-bde4-07b0daacef75.png)

Reply:
@AbdusSattar-70  
Thanks for your thorough review. I will clear out the input but I think the arrow is not a button in my website. It' s just a guide for the user to press enter and I think which is automatically understood by the users. And the previous code reviewers didn't have any problem at all for it. So thanks.

Reply:
Hello @sawmon71293,
Thank you for your response. Clearing out the input field is definitely a good idea and will help to improve the user experience. With regards to the arrow, if it's not intended to be a button, then it should be fine as is. However, if it's meant to indicate that the user should press enter to submit the form, it might be helpful to add some text or a label to indicate this. This can help users who may not be familiar with the convention of using enter instead of a mouse click to submit a form, so they can easily understand how to use this feature.